pandas/core/internals/managers.py

```diff
@@ -1474,21 +1387,123 @@ def take(self: T, indexer, axis: int = 1, verify: bool = True) -> T:
             consolidate=False,
         )
 
-    def _equal_values(self: T, other: T) -> bool:
+
+class BlockManager(_BlockManager):
```

Comment:
why don't we actually name it  Block2DManager then

Reply:
bc (to my chagrin) downstream packages access it